packages/mobile/src/utils/launchSelectImageActionSheet.ts

```diff
@@ -1,70 +1,62 @@
+import type { Image } from '@audius/common'
 import { ActionSheetIOS, Alert, Platform } from 'react-native'
-import type { ImageLibraryOptions, Asset } from 'react-native-image-picker'
-import { launchCamera } from 'react-native-image-picker'
+import type {
+  Image as CropPickerImage,
+  Options
+} from 'react-native-image-crop-picker'
 
-import type { Image } from 'app/types/image'
+import { store } from 'app/store'
 
-import launchImageLibrary from './launchImageLibrary'
-
-const photoOptions: ImageLibraryOptions = {
-  includeBase64: true,
-  maxWidth: 1440,
-  mediaType: 'photo',
-  quality: 0.9
-}
+import { openPicker, openCamera } from './launchImageLibrary'
+import { selectSystemTheme } from './theme'
 
 export const launchSelectImageActionSheet = (
-  callback: (image: Image, rawResponse: Asset) => void,
-  shareSheetTintColor: string,
+  onSelectImage: (image: Image) => void,
+  options?: Options,
   name?: string
```

Comment:
thoughts on renaming? `name` feels a bit ambiguous at first glance
```suggestion
  filename?: string
```

Reply:
its actually only for testing, so ill rename it to testID!